composer.json

```diff
@@ -12,17 +12,17 @@
     "minimum-stability": "stable",
     "require": {
         "yiisoft/yii2": "^2.0.15",
-        "psr/http-message": "^1.0",
+        "psr/http-message": "^1.0|^2.0",
         "psr/http-server-handler": "^1.0",
-        "laminas/laminas-diactoros": "^2.3",
-        "yidas/yii2-bower-asset": "2.0.13.1"
+        "laminas/laminas-diactoros": "^2.3|^3.0"
     },
     "require-dev": {
         "samdark/yii2-psr-log-target": "^1.0",
         "monolog/monolog": "~2.1.0",
         "squizlabs/php_codesniffer": "^3.2",
         "phpunit/phpunit": "^9.0",
-        "spiral/roadrunner": "^2.5"
+        "spiral/roadrunner": "^2.5",
+        "yidas/yii2-bower-asset": "2.0.13.1"
```

Comment:
Thanks for the PR! This dependency `yidas/yii2-bower-asset` I don't believe needs to be added. If you can remove it and update the composer.lock file I'd be happy to merge this in. I'd prefer not to lock this and let Yii2 automatically pull it in as needed.

Reply:
It was there before me, so I decided that you actually needed it and moved to the dev section. Should I still remove it? I believe without it you wouldn't be able to install the Yii2 framework since it requires some bower assets.

Reply:
@erickskrauch, @charlesportwoodii, I think, we should use `asset-packagist` (`bower-asset`) approach here, as it is now currently the main method for Yii2 applications. Otherwise, for now, this bridge will prevent `asset-packagist` from installing packages to `vendor/bower-asset` directory and use `yidas/yii2-bower-asset/bower` directory instead.

See for more information: https://www.yiiframework.com/doc/guide/2.0/en/structure-assets#bower-npm-assets